src/CoreBundle/Session/ChameleonSessionManager.php

```diff
@@ -125,10 +125,6 @@ public function boot()
         // add decorator to enforce write order (old data may not overwrite new data)
         $sessionHandler = new TPkgCmsSessionHandler_Decorator_EnforceWriteSequence($sessionStorage->getSaveHandler());
 
-        if ($this->metaDataTimeout > 0) {
-            $sessionHandler = new WriteCheckSessionHandler($sessionHandler);
```

Comment:
What was this doing (why can it be removed)?

Reply:
It avoided writing the session in case it wasn't changed. It now uses a core functionality in PHP. See https://symfony.com/blog/new-in-symfony-3-4-session-improvements#safer-and-lazier-sessions